### PR TITLE
Test Fixes second iteration

### LINKED ISSUE #17: Port language: axapta

### LINKED ISSUE #19: Port language: capnproto

### LINKED ISSUE #22: Port language: fsharp

### LINKED ISSUE #23: Port language: groovy

### LINKED ISSUE #28: Port language: monkey

### LINKED ISSUE #32: Port language: protobuf

### LINKED ISSUE #33: Port language: qml

### LINKED ISSUE #35: Port language: rsl

### PR DESCRIPTION
**Pull request by yescorp**
- Resolves #33 
- Resolves #17
- Resolves #19 
- Resolves #22 
- Resolves #23 
- Resolves #28 
- Resolves #32 
- Resolves #35 

- Failing tests in main branch: `62`
- Failing tests in current branch: `49`

### CODE REVIEW

**Comment by yescorp** [comment]
Languages that are currently not passing tests due to `Incorrect definition porting`:
1. `Bash`
2. `C++`
3. 
4. `Javascript` Also causes the following languages to fail, because they have it as a `sublanguage` dependency:
    -  `cos`
    -

File: flutter_highlighting/example/lib/main.dart [hunk 1]
```diff
@@ -3,6 +3,7 @@ import 'package:flutter_highlighting/flutter_highlighting.dart';
 import 'package:flutter_highlighting/theme_map.dart';
 import 'package:url_launcher/url_launcher.dart';
 import 'example_map.dart';
+import 'package:highlighting/highlighting_core.dart';
```
Comment (alexeyinkin) [inline flutter_highlighting/example/lib/main.dart:6]: Split.

File: highlighting/lib/highlighting_core.dart [hunk 2]
```diff
@@ -6,5 +6,6 @@ export 'package:highlighting/src/highlightv2.dart';
 export 'package:highlighting/src/mode.dart';
 export 'package:highlighting/src/node.dart';
 export 'package:highlighting/src/result.dart';
+export 'package:highlighting/languages/all.dart';
```
Comment (alexeyinkin) [inline highlighting/lib/highlighting_core.dart:9]: Undo.

File: highlighting/lib/src/extension/multi_class.dart [hunk 3]
```diff
@@ -9,7 +9,7 @@ final MultiClassError = Exception('MultiClass Error');
 /// `key` - `"beginScope" | "endScope"`
 void remapScopeNames(
   Mode mode,
-  List<dynamic> regexes,
+  List<Pattern> regexes,
```
Comment (alexeyinkin) [inline highlighting/lib/src/extension/multi_class.dart:12]: /// [regexes] -- A list of [RegExp] or strings to be compiled to [RegExp].

**Review by alexeyinkin — CHANGES_REQUESTED** [review]

**Review by alexeyinkin — APPROVED** [review]

**Pull request merged**